src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,73 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A task that stores the deadline.
+ */
+public class Deadline extends Task {
+    /**
+     * The date of the deadline.
+     */
+    private LocalDate date;
+
+    /**
+     * The time of the deadline.
+     */
+    private LocalTime time;
+
+    /**
+     * A constructor used to initialize the deadline.
+     *
+     * @param description the description of the deadline.
+     * @param date the date of the deadline
+     * @param time the time of the deadline
+     */
```

Comment:
Think you didnt have any line spacing between the description and @param in your newer files too :)) might wanna standardise the documentation?